README.md

```diff
@@ -46,14 +46,23 @@ We use SASS CSS pre-compiler. If you're doing front-end work your local machine
 ### Installing
     $ git clone https://github.com/uktrade/directory-ui-export-readiness
     $ cd directory-ui-export-readiness
-    $ virtualenv .venv -p python3.5
+    $ virtualenv .venv -p python3.6
     $ source .venv/bin/activate
-    $ pip install -r requirements_test.txt
+    $ python -m  pip install -r requirements_test.txt
 
 ### Running the webserver
     $ source .venv/bin/activate
     $ make debug_webserver
 
+### Running Redis Server
+    $ brew install redis
+    $ redis-server
+warning: This installs the latest version of Redis server on local machine not the pinned version.
+
+### Setting enviroment variables for running the web server
+  Locate from the repo file 'makefile' and within the bash prompt run all the export variables from 'DEBUG_SET_ENV_VARS'  &  'TEST_SET_ENV_VARS'
+
```

Comment:
this will cause clashes between env vars on different services. e.g, SSO, API, and CMS all have an env var called `DATABASE_URL`

Instead call the `Makefile` commands such as. This will handle settings the env vars for us without the clashing occuring:
* `make debug_webserver` - runs the django webserver
* `make debug_test` - run the tests
* make debug_manage cmd="<command>" - run an arbitrary management command

Feel free to update the readme to suggest that the dev becomes familar with the `Makefile` commands